### ISSUE TITLE
ALZ: "how many tenants" not phrased as recommendation

### ISSUE THREAD

**Issue by erjosito** [Azure/review-checklists#478]
Item with GUID 70c15989-c726-42c7-b0d3-24b7375b9201, "How Many Azure AD Tenants do you manage?" is not actually phrased as a recommendation. The related link sends the user to the multitenancy architecture guidance, which is not obvious from the check text.

Maybe we can rephrase this as "Unless you have a requirement for multitenancy, more than one Entra tenant will add complexity to your environment"?

**Comment by tsc-buddy** [comment]
@erjosito - thanks for that. updated is in the linked PR.
